### PR TITLE
Enable and stabilise e2e tests for Swaps, add debouncing

### PR DESCRIPTION
**Pull request by dan437**
## Description
This PR: 
- enables Swaps E2E tests
- adds debouncing for the `Swap from` and `Swap to` fields
- cleans up console errors in Swaps

### CODE REVIEW

File: test/e2e/swaps/shared.js [hunk 1]
```diff
@@ -8,6 +10,17 @@ const ganacheOptions = {
...
 
+const driverOptions = {
+  timeOut: 20000,
+};
+
+const withFixturesOptions = {
+  fixtures: new FixtureBuilder().build(),
+  ganacheOptions,
+  driverOptions,
+  failOnConsoleError: false,
```
Comment (PeterYinusa) [inline test/e2e/swaps/shared.js:21]: is failOnConsoleError still needed?

File: test/e2e/swaps/shared.js [hunk 2]
```diff
@@ -28,19 +41,27 @@ const buildQuote = async (driver, options) => {
   await driver.wait(async () => {
     const tokens = await driver.findElements('.searchable-item-list__item');
     return tokens.length > 1;
-  }, 10000);
+  }, 20000);
```
Comment (PeterYinusa) [inline test/e2e/swaps/shared.js:44]: Why do we need to wait this long?

File: test/e2e/swaps/shared.js (hunk 1 above)
Comment (dan437) [reply test/e2e/swaps/shared.js:21]: Yes, we are still getting React errors with some tests:
image

I can try to clean up those errors and then this won't be needed.

File: test/e2e/swaps/shared.js (hunk 2 above)
Comment (dan437) [reply test/e2e/swaps/shared.js:44]: I've increased it just to be on the safe side if token loading would get little longer than 10s.

File: test/e2e/swaps/shared.js [hunk 3]
```diff
@@ -8,6 +10,16 @@ const ganacheOptions = {
   ],
 };
 
+const driverOptions = {
+  timeOut: 20000,
+};
```
Comment (PeterYinusa) [inline test/e2e/swaps/shared.js:15]: can we get rid of this?

**Review by PeterYinusa — DISMISSED** [review]

**Review by PeterYinusa — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**